<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="5" failed="0" ignored="1" total="6" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="235" started-at="2018-12-31T08:32:58Z" finished-at="2018-12-31T08:32:58Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="235" started-at="2018-12-31T08:32:58Z" finished-at="2018-12-31T08:32:58Z">
      <class name="com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries">
        <test-method status="SKIP" signature="beforeTest()[pri:0, instance:com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries@887af79]" name="beforeTest" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="SKIP" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries@887af79]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="InvalidDOB(java.util.Map)[pri:0, instance:com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries@887af79]" name="InvalidDOB" duration-ms="0" started-at="2018-12-31T14:02:58Z" description="Verifying the DL process configuration with Invalid DOB" data-provider="InvalidDOB" finished-at="2018-12-31T14:02:58Z">
          <exception class="com.codepine.api.testrail.TestRailException">
            <message>
              <![CDATA[401 - External auth: Bind: Invalid credentials]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.codepine.api.testrail.TestRailException: 401 - External auth: Bind: Invalid credentialsat com.codepine.api.testrail.TestRailException$Builder.build(TestRailException.java:57)at com.codepine.api.testrail.Request.execute(Request.java:142)at com.uiFramework.ust.experian.utils.TestRailReport.createTestrailRun(TestRailReport.java:71)at com.uiFramework.ust.experian.testbase.TestBase.beforeSuite(TestBase.java:59)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- com.codepine.api.testrail.TestRailException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InvalidDOB -->
        <test-method status="SKIP" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries@887af79]" name="afterMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="SKIP" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries@887af79]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="ValidDLCheck(java.util.Map)[pri:0, instance:com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries@887af79]" name="ValidDLCheck" duration-ms="0" started-at="2018-12-31T14:02:58Z" description="Verifying the DL process configuration with Valid DL number" data-provider="ValidDLCheck" finished-at="2018-12-31T14:02:58Z">
          <exception class="com.codepine.api.testrail.TestRailException">
            <message>
              <![CDATA[401 - External auth: Bind: Invalid credentials]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.codepine.api.testrail.TestRailException: 401 - External auth: Bind: Invalid credentialsat com.codepine.api.testrail.TestRailException$Builder.build(TestRailException.java:57)at com.codepine.api.testrail.Request.execute(Request.java:142)at com.uiFramework.ust.experian.utils.TestRailReport.createTestrailRun(TestRailReport.java:71)at com.uiFramework.ust.experian.testbase.TestBase.beforeSuite(TestBase.java:59)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- com.codepine.api.testrail.TestRailException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidDLCheck -->
        <test-method status="SKIP" signature="afterTest()[pri:0, instance:com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries@887af79]" name="afterTest" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="SKIP" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries@887af79]" name="afterMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- com.uiFramework.ust.experian.testScripts.DrivingLicenseQueries -->
      <class name="com.uiFramework.ust.experian.testScripts.BankWiazrdQueries">
        <test-method status="FAIL" signature="beforeSuite()[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="beforeSuite" is-config="true" duration-ms="1262" started-at="2018-12-31T14:02:56Z" finished-at="2018-12-31T14:02:58Z">
          <exception class="com.codepine.api.testrail.TestRailException">
            <message>
              <![CDATA[401 - External auth: Bind: Invalid credentials]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.codepine.api.testrail.TestRailException: 401 - External auth: Bind: Invalid credentialsat com.codepine.api.testrail.TestRailException$Builder.build(TestRailException.java:57)at com.codepine.api.testrail.Request.execute(Request.java:142)at com.uiFramework.ust.experian.utils.TestRailReport.createTestrailRun(TestRailReport.java:71)at com.uiFramework.ust.experian.testbase.TestBase.beforeSuite(TestBase.java:59)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- com.codepine.api.testrail.TestRailException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="SKIP" signature="beforeTest()[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="beforeTest" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="SKIP" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="BWAccClosed(java.util.Map)[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="BWAccClosed" duration-ms="0" started-at="2018-12-31T14:02:58Z" description="Verifying the BWAccClosed process configuration" data-provider="BWAccClosed" finished-at="2018-12-31T14:02:58Z">
          <exception class="com.codepine.api.testrail.TestRailException">
            <message>
              <![CDATA[401 - External auth: Bind: Invalid credentials]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.codepine.api.testrail.TestRailException: 401 - External auth: Bind: Invalid credentialsat com.codepine.api.testrail.TestRailException$Builder.build(TestRailException.java:57)at com.codepine.api.testrail.Request.execute(Request.java:142)at com.uiFramework.ust.experian.utils.TestRailReport.createTestrailRun(TestRailReport.java:71)at com.uiFramework.ust.experian.testbase.TestBase.beforeSuite(TestBase.java:59)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- com.codepine.api.testrail.TestRailException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BWAccClosed -->
        <test-method status="SKIP" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="afterMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="SKIP" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="BWAccClosed(java.util.Map)[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="BWAccClosed" duration-ms="0" started-at="2018-12-31T14:02:58Z" description="Verifying the BWAccClosed process configuration" data-provider="BWAccClosed" finished-at="2018-12-31T14:02:58Z">
          <exception class="com.codepine.api.testrail.TestRailException">
            <message>
              <![CDATA[401 - External auth: Bind: Invalid credentials]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.codepine.api.testrail.TestRailException: 401 - External auth: Bind: Invalid credentialsat com.codepine.api.testrail.TestRailException$Builder.build(TestRailException.java:57)at com.codepine.api.testrail.Request.execute(Request.java:142)at com.uiFramework.ust.experian.utils.TestRailReport.createTestrailRun(TestRailReport.java:71)at com.uiFramework.ust.experian.testbase.TestBase.beforeSuite(TestBase.java:59)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- com.codepine.api.testrail.TestRailException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BWAccClosed -->
        <test-method status="SKIP" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="afterMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="SKIP" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="BWAccHolderDeceased(java.util.Map)[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="BWAccHolderDeceased" duration-ms="0" started-at="2018-12-31T14:02:58Z" description="Verifying the BWAccHolderDeceased process configuration" data-provider="BWAccHolderDeceased" finished-at="2018-12-31T14:02:58Z">
          <exception class="com.codepine.api.testrail.TestRailException">
            <message>
              <![CDATA[401 - External auth: Bind: Invalid credentials]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.codepine.api.testrail.TestRailException: 401 - External auth: Bind: Invalid credentialsat com.codepine.api.testrail.TestRailException$Builder.build(TestRailException.java:57)at com.codepine.api.testrail.Request.execute(Request.java:142)at com.uiFramework.ust.experian.utils.TestRailReport.createTestrailRun(TestRailReport.java:71)at com.uiFramework.ust.experian.testbase.TestBase.beforeSuite(TestBase.java:59)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- com.codepine.api.testrail.TestRailException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BWAccHolderDeceased -->
        <test-method status="SKIP" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="afterMethod" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="SKIP" signature="afterTest()[pri:0, instance:com.uiFramework.ust.experian.testScripts.BankWiazrdQueries@7b49cea0]" name="afterTest" is-config="true" duration-ms="0" started-at="2018-12-31T14:02:58Z" finished-at="2018-12-31T14:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.uiFramework.ust.experian.testScripts.BankWiazrdQueries -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
